Library/Formula/vim.rb

```diff
@@ -35,6 +35,15 @@ class Vim < Formula
   conflicts_with "ex-vi",
     :because => "vim and ex-vi both install bin/ex and bin/view"
 
+  # Default requirements mean installing the bottle here requires
+  # installing our Perl, Python & Ruby. To prevent that for everyone
+  # just pour the bottle when requirements already satisfied.
+  def pour_bottle?
+    (which("python") == HOMEBREW_PREFIX/"bin/python") &&
+      (which("ruby") == HOMEBREW_PREFIX/"bin/ruby") &&
+      Formula["perl"].installed?
+  end
```

Comment:
Ah, I see what you're doing here actually, ignore me, I was being stupid. Still, it feels a bit weird that this bottle will only be used if you've installed three system duplicates. I'd be tempted to say it should be the other way around (e.g. build against system versions by default and don't use the bottle if you're using non-system versions).